### DISCUSSION TITLE
Compilation auto de la documentation

### DISCUSSION THREAD

**Discussion by spelhate** [mviewer/mviewer#560] [Announcements]
La compilation de la documentation s'effectue à nouveau de façon automatique. Ainsi la doc latest https://mviewerdoc.readthedocs.io/fr/latest/search.html?q=pattern&check_keywords=yes&area=default intègre bien la PR mergée il y a quelque minutes sans action particulière côté readthedocs.

**Comment by Gaetanbrl** [comment]
Tu as mis en place quelque chose dans le projet github ?

**Reply by Gaetanbrl** [reply]
Je vois qu'il  y a un webhook
https://github.com/geobretagne/mviewer/settings/hooks

**Comment by spelhate** [comment]
Je n'ai rien fait côté mviewer. J'ai juste resynchronisé le web hook côté readthedocs

**Reply by Gaetanbrl** [reply]
c'est ca qui a du remettre le build auto, car je crois qu'il n'y avait pas de webhook lors du dernier PSC (vu que ca devait être sur le repo de la doc).
En tout cas le build auto c'est top ! :) Merci
